README.md

```diff
@@ -419,6 +484,20 @@ const raygunClient = new raygun.Client().init({
 
 Including column numbers can enable source mapping if you have minified or transpiled code in your stack traces.
 
+### APM Bridge Setup
+
+Raygun4Node can be used in conjunction with Raygun APM for Node.js.
+
+Follow the instructions on https://raygun.com/platform/apm to set up your application.
+
+Then, call to `setApmBridge()` to connect both packages:
+
+```js
+raygunClient.setApmBridge(require("raygun-apm/lib/src/crash_reporting"));
```

Comment:
That looks like a nasty setup. Can we make it without arguments and infer it? Still have the explicit call with the argument as fallback

Reply:
Can you take a look at the PR where we introduced this and review it? https://github.com/MindscapeHQ/raygun4node/pull/222